src/raygun.rum.js

```diff
@@ -39,7 +40,12 @@ var raygunRumFactory = function(window, $, Raygun) {
     this.excludedUserAgents = excludedUserAgents;
     this.maxVirtualPageDuration = maxVirtualPageDuration || 1800000; // 30 minutes
     this.ignoreUrlCasing = ignoreUrlCasing;
-    this.customTimingsEnabled = customTimingsEnabled;
+    /**
+     * Note: the `customTimingsEnabled` flag is for tracking legacy custom timings
+     * because that api prevents page timings from being sent until the main request is completed
+     */
+    this.customTimingsEnabled = customTimingsEnabled; 
+    this.automaticPerformanceCustomTimings = automaticPerformanceCustomTimings;
```

Comment:
Just to confirm: The new custom timings will still block network requests from being sent until the timing has fully resolved is that right?

Reply:
Spoke in person :) 

But for those watching the new variant of custom timings does not block network requests.

Reply:
Have spoken in person about this one.